content/cybersecurity/concepts/spear-phishing/spear-phishing.md

```diff
@@ -0,0 +1,32 @@
+# Spear phishing
```

Comment:
We're missing the metadata for this entry. Look at the [source here](https://github.com/Codecademy/docs/blob/main/content/cybersecurity/concepts/cyber-attack/cyber-attack.md) to see an example of what we need.

Also, remove the h1 header, the metadata title will provide that.